functionalTest/test/utilities/reusableLibrary.js

```diff
@@ -1,6 +1,21 @@
-/**
- * Created this file to include all reusable methodes in future.
- */
-class reusableLibrary {}
+import dashboard from "../pageobjects/dashboardPage";
 
+class reusableLibrary {
+  sliderTrackResult() {
+    dashboard.selectTimeSpan("ALL").click();
+    expect(dashboard.sliderTrack).toHaveAttr("style", "left: 0%;");
+
+    dashboard.selectTimeSpan("Last 3M").click();
+    expect(dashboard.sliderTrack).toHaveAttr("style", "left: 60.6838%;");
```

Comment:
These percentages will break as soon as the dataset contains more days. Either we figure out a method now to set a particular dataset for a test, or change these to be more robust to data changes. Options for the latter are to:

1. Inspect other attributes of the slider (.MuiSlider-root .MuiSlider-thumb attr("aria-valuenow"))
2. Just live with calculated proportional values for these attributes ie 3 months should be approximately 3 times as wide as 1 month, etc

Neither are particularly nice

Reply:
- Tried with  attributes (.MuiSlider-root .MuiSlider-thumb attr("aria-valuenow")) but test was failing. Unable to find any robust solution for slidetrack validation, so changed the expect condition to only 'Tobedisplayed()'. 
- checked with second solution for width attribute but proportion doesn't match with calculated result. (small difference in actual and calculated values)

Reply:
We can debug the first solution if you have time. Alternatively, your second solution (manually calculating the expected proportion) could still work. It doesn't have to be exact (without calculating pixels used in padding, margins, etc). We use expects with margins of error.

Reply:
@donal-s Yes, I will need your help to work on this solution. Please let me know when we can take a call? :)

Reply:
Worked on a solution.